index.js

```diff
@@ -0,0 +1,109 @@
+// random 숫자 만들기
+let numberArr = [1, 2, 3, 4, 5, 6, 7, 8, 9]; // 숫자 값 비교를 위한 숫자배열
+
+let randomArr = []; // 게임시작 시, 세팅되는 랜덤숫자배열 ex) [1,2,3]
+let wrongCount = 0; // 잘못된 횟수
+
+// Get Dom
+const resultElement = document.getElementById("result"); // 결과가 나오는 span tag
+const inputElement = document.getElementById("input-number"); // 사용자가 숫자를 입력하는 input tag
+const startButtonElement = document.getElementById("start-btn"); // 게임 시작 시, 랜덤 숫자배열 생성
+const enterButtonElement = document.getElementById("enter-btn"); // 사용자 입력 enter
+const restartButtonElement = document.getElementById("restart-btn"); // 재시작 버튼
+
+function createRandomArray() {
+  console.log("createRandomArray");
+  randomArr = [];
+  for (let i = 0; i < 3; i++) {
+    // let draw = numberArr.pop();
+    // let draw = numberArr.shift();
+    let draw = numberArr.splice(Math.floor(Math.random() * (9 - i)), 1)[0];
+    randomArr.push(draw);
+  }
+
+  //console.log(randomArr);
+}
+
+function initializeInput() {
+  inputElement.value = "";
+  inputElement.focus();
+}
+
+function gameStart() {
```

Comment:
일반적으로 함수 이름은 동사로 시작합니다.
함수는 행위를 하기 때문입니다.
```suggestion
function startGame() {
```